bootstrap/16-files.sh

```diff
@@ -0,0 +1,29 @@
+function install_recur() {
+    for i in "$1"/*
+    do
+        full_path=$i
+        relative_path=${i#$CUSTOM_FILES}
```

Comment:
Use `local` for local variables. It helps to avoid polluting the global name space.